src/services/WorkPeriodPaymentService.js

```
@@ -55,47 +56,29 @@ async function _createSingleWorkPeriodPayment (workPeriodPayment, createdBy) {
  * @returns {Object} the created workPeriodPayment
  */
 async function _createSingleWorkPeriodPaymentWithWorkPeriodAndResourceBooking (workPeriodPayment, createdBy, correspondingWorkPeriod, correspondingResourceBooking) {
-  if (!correspondingResourceBooking.billingAccountId) {
-    throw new errors.ConflictError(`id: ${correspondingWorkPeriod.resourceBookingId} "ResourceBooking" Billing account is not assigned to the resource booking`)
+  if (_.isNil(correspondingResourceBooking.billingAccountId)) {
```

We should also return an error if `memberRate` is `0`.